samples/bluetooth/mesh/sensor_server/src/model_handler.c

```diff
@@ -317,6 +329,26 @@ static struct bt_mesh_sensor rel_chip_temp_runtime = {
 	},
 };
 
+static int people_count_get(struct bt_mesh_sensor_srv *srv,
+				 struct bt_mesh_sensor *sensor,
+				 struct bt_mesh_msg_ctx *ctx,
+				 struct bt_mesh_sensor_value *rsp)
+{
+	int err =
+		bt_mesh_sensor_value_from_micro(sensor->type->channels[0].format,
+						BASE_UNITS_TO_MICRO(dummy_people_count_value), rsp);
+
+	if (err && err != -ERANGE) {
+		printk("Error encoding people count (%d)", err);
+	}
+	return err;
```

Comment:
This should be inside the `if`, and the function should return 0 at the end. This is because when it ignores `-ERANGE`, it shouldn't return it as an error either. Same in other getters.